src/fontra/client/lang/ja.js

```diff
@@ -220,7 +220,9 @@ export const strings = {
   "menubar.view.find-glyphs-that-use": "Find glyphs that use '%0'",
   "menubar.view.remove-selected-glyph-from-canvas": "Remove selected glyph from canvas",
   "menubar.view.replace-selected-glyph-on-canvas": "Replace selected glyph on canvas",
+  "menubar.view.select-next-glyph": "Select next glyph",
```

Comment:
So, the reason is, that these keys are actually really missing in main branch for Japanese. 
'Merge branch 'main' into issue-1752-missing-translation-keys' does not include Japanese, but the keys for previous and next glyph are part of it. Then one commit later 'Merge pull request [#1764] from googlefonts/issue-1752-missing-translation-keys' includes Japanese, but without keys for previous/next glyph.
I suspect, this is due to 'WIP'. During my work on previous/next glyphs the keys work marked as WIP in the google sheet. It was wart of the PR, because I updated the languages files with `--include-wip`. I removed 'WIP' from the google sheet some time (maybe one day) after we merged it the PR.

Reply:
I am closing this, as this is not an issue related to this PR.